trade-finance-manager-ui/server/api-response-types/utilisation-report-reconciliation-details-response-body.ts

```diff
@@ -27,7 +27,7 @@ export type KeyingSheetRow = {
   feePayments: {
     currency: Currency;
     amount: number;
-    dateReceived: IsoDateTimeStamp;
+    dateReceived: IsoDateTimeStamp | undefined;
```

Comment:
I think this should now be `IsoDateTimeStamp | null` (probably the same for the tfm-ui response type)